meinberlin/assets/scss/components_user_facing/_alert.scss

```diff
@@ -1,26 +1,30 @@
-@use "sass:color";
+.alert {
+    margin: $padding 0;
+    padding: $padding * 0.625;
+    background-color: $message-light-blue;
+    color: $text-base;
+    text-align: start;
+}
 
-$messages-margin-bottom: 25px;
+.alert__content {
+    padding: $padding $padding * 0.5;
+}
 
-.messages {
-    margin: 0 0 $messages-margin-bottom;
+.alert__headline {
+    margin: 0;
     padding: 0;
-
-    li {
-        display: block;
-    }
-
-    + .messages {
-        margin-top: -$messages-margin-bottom;
-    }
+    color: inherit;
 }
 
-.alert {
+.alert__text {
     margin: 0;
-    padding: $padding * 1.2 0;
-    background-color: $message-light-blue;
-    color: $text-base;
-    text-align: center;
+    padding: 0;
+}
+
+.alert__close {
+    padding: 0;
+    float: right;
```

Comment:
i'd go for flexbox here. It's more responsive and avoids potential layout issues.